src/Greenshot/Languages/language-de-DE.xml

```diff
@@ -219,29 +219,29 @@ Bitte überprüfen Sie die Schreibrechte oder wählen Sie einen anderen Speicher
 		<resource name="settings_destination_picker">Ziel dynamisch auswählen</resource>
 		<resource name="settings_destination_printer">An Drucker senden</resource>
 		<resource name="settings_editor">Editor</resource>
-		<resource name="settings_filenamepattern">Dateiname-Muster</resource>
+		<resource name="settings_filenamepattern">Dateinamensmuster</resource>
 		<resource name="settings_general">Allgemein</resource>
 		<resource name="settings_iecapture">Internet Explorer abfotografieren</resource>
 		<resource name="settings_jpegquality">JPEG-Qualität</resource>
 		<resource name="settings_language">Sprache</resource>
 		<resource name="settings_message_filenamepattern">Die folgenden Platzhalter werden im festgelegten Muster automatisch ersetzt:
-${YYYY} Jahr, 4-stellig
-${MM} Monat, 2-stellig
-${DD} Tag, 2-stellig
-${hh} Stunde, 2-stellig
-${mm} Minute, 2-stellig
-${ss} Sekunde, 2-stellig
-${NUM} inkrementierende Zahl, 6-stellig
-${title} Fenstertitel
-${user} Windows-Benutzername
-${domain} Windows-Domäne
-${hostname} Computername
+${YYYY}		Jahr, 4-stellig
+${MM}		Monat, 2-stellig
+${DD}		Tag, 2-stellig
+${hh}		Stunde, 2-stellig
+${mm}		Minute, 2-stellig
+${ss}		Sekunde, 2-stellig
+${NUM}		inkrementierende Zahl, 6-stellig
+${title}		Fenstertitel
+${user}		Windows-Benutzername
+${domain}	Windows-Domäne
+${hostname}	Computername
 
 Greenshot kann auch Verzeichnisse dynamisch erstellen.
-Verwenden Sie den Backslash \ um Verzeichnisse vom Dateinamen zu trennen.
-Zum Beispiel: ${YYYY}-${MM}-${DD}\${hh}-${mm}-${ss}
-Dieses Muster legt ein Verzeichnis für den aktuellen Tag im Standard-Speicherort an und speichert die Bilddateien im Uhrzeit-Format, mit der vorgegebenen Dateiendung ab.
-z.B. C:\Users\MaxMustermann\Desktop\2012-08-13\12-58-32.png</resource>
+Verwenden Sie den Backslash \, um Verzeichnisse vom Dateinamen zu trennen.
+Beispiel: ${YYYY}-${MM}-${DD}\${hh}-${mm}-${ss}
+Dieses Muster legt ein Verzeichnis für den aktuellen Tag im Standard-Speicherort an und speichert die Bilddateien im Uhrzeit-Format mit der vorgegebenen Dateiendung ab.
+Z. B.: C:\Users\MaxMustermann\Desktop\2012-08-13\12-58-32.png</resource>

```

Comment:
Who uses `Z. B.:`..?

Reply:
OK,
Beispiel: C:\Users\MaxMustermann\Desktop\2012-08-13\12-58-32.png</resource>
sounds better (like 2 lines before)

Reply:
Hard to say, since if this is a block text it may make more sense to use `z.B.` as it feels more natural. 

Some of your suggestions I agree with, others seem mostly a matter of preference.

Reply:
Even though `Z. B.` looks weird I think it is correct in this context. But I really like the idea of just using `Beispiel` (the abbreviation does not save so many characters anyway 😁 )